doc/examples/features_detection/plot_brief.py

```diff
@@ -54,8 +51,23 @@
 
 plot_matches(ax[0], img1, img2, keypoints1, keypoints2, matches12)
 ax[0].axis('off')
+ax[0].set_title("Original Image VS Transformed Image")
+print('Transformed Image using AffineTransform parameters:')
```

Comment:
Same here, I don't see how this logging information improves the example. Could you elaborate, please?